astropy/coordinates/tests/accuracy/test_altaz_icrs.py

```diff
@@ -0,0 +1,145 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Accuracy tests for AltAz to ICRS coordinate transformations.
+
+We use "known good" examples computed with other coordinate libraries.
+
+Note that we use very low precision asserts because some people run tests on 32-bit
+machines and we want the tests to pass there.
+TODO: check if these tests pass on 32-bit machines and implement
+higher-precision checks on 64-bit machines.
+"""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from ....tests.helper import pytest, catch_warnings
+from .... import units as u
+from ....time import Time
+from ...builtin_frames import AltAz
+from ... import EarthLocation
+from ... import Angle, SkyCoord
+
+
+# Observatory position for `kpno` from here:
+# http://idlastro.gsfc.nasa.gov/ftp/pro/astro/observatory.pro
+kpno = EarthLocation(lon=Angle('111d36.0m'),
```

Comment:
And in real life too, correct? From Wikipedia's article on Longitude:

*Longitude is given as an angular measurement ranging from 0° at the Prime Meridian to +180° eastward and −180° westward*